plugins/csp-measurement-tools/src/voronoi/Breakpoint.hpp

```diff
@@ -24,15 +24,15 @@ class Breakpoint {
   Vector2f const& position() const;
   Edge            finishEdge(Vector2f const& end) const;
 
-  Arc *       mLeftArc, *mRightArc;
-  Breakpoint *mLeftChild, *mRightChild, *mParent;
+  Arc *       mLeftArc{nullptr}, *mRightArc{nullptr};
+  Breakpoint *mLeftChild{nullptr}, *mRightChild{nullptr}, *mParent{nullptr};
```

Comment:
This is a rule we enabled, but clang-tidy can't apply it to member variables due to technical limitations. Especially with the use of pointer we should enforce that.
```suggestion
  Arc* mLeftArc{nullptr};
  Arc* mRightArc{nullptr};
  
  Breakpoint* mLeftChild{nullptr};
  Breakpoint* mRightChild{nullptr};
  Breakpoint* mParent{nullptr};
```